_includes/appcast.inc

```diff
@@ -5,12 +5,20 @@
         <description>Most recent changes with links to updates.</description>
         <language>en</language>
         {% for release in site.github.releases %}
-            {% if release.prerelease and page.prereleases %}{% break %}{% endif %}
-            {% assign tag_name = release.tag_name %}
+            {% if release.prerelease and !page.include_prereleases %}{% continue %}{% endif %}
+
+            {% comment %}
+            <!--
+                This was all related to the "new" build system referenced in
+                c6891a8e (ca 2018) and seems to want to parse tags like v/0.17.
```

Comment:
What's about to add complete sha1 url `https://github.com/gitx/gitx.github.io/commit/c6891a8e044d4` ?

Btw, the same commit is in main repo [https://github.com/gitx/gitx/commit/c6891a8e044d4. It's confusing why @tiennou push this complete repo in gitx/gitx as branch with unrelated history. Was it an accident ?